doc/api/modules.md

```diff
@@ -552,7 +552,20 @@ added: v0.1.13
 
 * {Function}
 
-To require modules.
+Used to import modules, `JSON` and local files. Modules can be imported from `node_modules`. Local modules and JSON files can be imported using the pattern `'./`.
+
+Examples:
+
+```js
+// importing localModule
+const myModule = require('./path/myLocalModule');
+
+// importing localModule
+const jsonData = require('./path/filename.json');
+
+// importing internal module from node_modules or node.js internals
```

Comment:
I'd remove 'internal' (so just `importing module`) as it may be confusing because modules in node_modules are actually external (opposed to your modules and node core). Wdyt?

Reply:
node.js -> Node.js :)